### PR TITLE
Add the ability to roundtrip class custom fields from the database

### PR DESCRIPTION
**Pull request by obi1kenobi**

### CODE REVIEW

File: pyorient/ogm/graph.py [hunk 1]
```diff
@@ -182,6 +182,8 @@ def extract_properties(property_schema, is_edge):
             is_edge = bases and bases[0].decl_type == DeclarativeType.Edge
             props.update(extract_properties(class_def['properties'], is_edge))
 
+            props['class_fields'] = class_def.get('customFields', None) or {}
```
Comment (obi1kenobi) [inline pyorient/ogm/graph.py]: `.get()` has an implied `None` for a second parameter

File: pyorient/ogm/graph.py [hunk 2]
```diff
@@ -326,6 +328,12 @@ def create_class(self, cls):
                     # Index already exists
                     pass
 
+        # Store custom fields
+        for name, value in getattr(cls, 'class_fields', {}).iteritems():
+            self.client.command(
+                    'ALTER CLASS {} CUSTOM {}={}'
```
Comment (obi1kenobi) [inline pyorient/ogm/graph.py]: I don't like this, let's not add this functionality until it's necessary. Unsanitized SQL makes me very unhappy.

**Comment by obi1kenobi** [comment]
I squashed the commits together, and pending successful CI am willing to merge this and open a PR to the main repo. Does that sound good?

**Comment by lodrion** [comment]
sounds fine

**Pull request merged**